### PR TITLE
Add test for empty observables and make some inprovements

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: tests/functional/tests/test_empty_observables.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        **{'headers': module_headers}
+    )
+
+    pulsedive_data = response_from_all_modules['data']
+
+    response_from_pulsedive = get_observables(
+        pulsedive_data, MODULE_NAME)
+
+    # check that we have only one response from Pulsedive
+    assert type(response_from_pulsedive) == dict, 'Got 2 entity from Pulsedive'
```
Comment (AnastasiiaRozlyvan) [inline tests/functional/tests/test_empty_observables.py:90]: I think it's better to use the plural form for the word 'entity'. Something like: "Got 2 entities from Pulsedive"

Comment (mistergreen22) [reply tests/functional/tests/test_empty_observables.py]: okey)

**Review by nonspher — APPROVED** [review]

**Review by AnastasiiaRozlyvan — APPROVED** [review]
